nucliadb/src/nucliadb/ingest/orm/resource.py

```diff
@@ -166,6 +167,13 @@ async def set_basic(
 
             self.basic.MergeFrom(payload)
 
+            # Make sure title is cleared if the flag is set
+            if self.basic.title == CLEAR_TITLE_FLAG:
+                # We set the title to the resource UUID so that it is not empty.
+                # If there is a computed title in the link or file extracted metadata
+                # by processing, it will be updated later.
+                self.basic.title = self.uuid
```

Comment:
If this is actually setting the title to the UUID, maybe we could avoid the `CLEAR_TITLE_FLAG` and set the title directly to uuid in the message to index? Just a thought, not sure if it's actually a good idea.